### DISCUSSION TITLE
only low RPM after arm

### DISCUSSION THREAD

**Discussion by petrweigl** [iNavFlight/inav#9995] [Q&A]
On my quadcopter, I sometimes experience an issue after multiple takeoffs and landings, where I always turn off and then rearm the motors between flights. The motors do not respond properly to the throttle upon the next start, spinning only at low RPMs, and sometimes only two out of four propellers are spinning. I'm using ESCs with the DSHOT 300 protocol and BLHeli_S firmware, and these problems occur even with a sufficiently charged battery. Do you have any tips on how to resolve this issue or any ideas on what could be causing it? (inav 7.1, FC speeedebee f405 V4, ANGLE MODE). Thank you.

**Comment by Jetrell** [comment]
A couple of things to ask you.

Do you hear the ESCs beep their melody after disarming, before you rearm without a power cycle ?
Do you have STATS enabled ?
Have you tried changing Dshot speed ?  i.e. 150 or 600.. (long story)

**Reply by petrweigl** [reply]
1/ there is one short beep after dearm
2/ STATS do you mean STATE FLAGS in log? Or something else, please.
3/ I did not tried other Dshot Speed yet , but I will if it is necesarry. But on another copter - same ESC, same FC, same setting - just another KV motors it WORKS well 

Edit: I chanded to d600 and d150 - but  still the same behaviour. Only low rpm reached. I also check logfile and there is for example rcCommand throttle 2000, motor output 100% but visually I can see that the motor is spining slowly. Copter is of course still sitting on the ground.

**Reply by petrweigl** [reply]
![stat](https://github.com/iNavFlight/inav/assets/16215709/762ecafa-298d-4cc5-98d0-8fe465b5f75a)

So you probably mean these STATS. So i will enable it.

**Reply by Jetrell** [reply]
No. Don't enable STATS.. It can cause issues for multirotors.. I was only asking to see if it could have been the cause.
